Scripts/build-phases/generate-credentials.sh

```diff
@@ -62,6 +62,16 @@ else
     ##
     mkdir -p ${DERIVED_PATH}
 
+    if which rbenv; then
+      # Fix an issue where, depending on the shell you are using on your machine and your rbenv setup,
+      #   running `ruby` in a bash script from Xcode script build phase might not use the right ruby
+      #   (and thus not find the appropriate gems installed by bundle & Gemfile.lock and crash).
+      # So if rbenv is installed, make sure the shims for `ruby` are too in the context of this bash script,
+      #   so that it uses the right ruby version defined in `.ruby-version` instead of risking to use the system one.
+      eval "$(rbenv init -)"
+      rbenv rehash
+    fi
+
```

Comment:
I added this patch in order to be able to compile, otherwise I had a failure on the build phase when running it on my machine. Not sure why it worked in the past back when I was still working on (and compiling) WCiOS once in a while as RN and failed today, but I figured it probably doesn't hurt anyway?

Would love to find someone who might not have rbenv setup (at all, or in a different way that I did with by `.zshrc`) to check if this doesn't break _their_ setup